bindings/go/examples/gas_station/main.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	sdk "bindings/iota_sdk_ffi"
+	"fmt"
+	"log"
+)
+
+func main() {
+	client := sdk.GraphQlClientNewLocalnet()
+	gasStationUrl := "http://0.0.0.0:9527"
+	keypair := sdk.Ed25519PrivateKeyGenerate()
+	sender := keypair.PublicKey().DeriveAddress()
+	simpleKey := sdk.SimpleKeypairFromEd25519(keypair)
+
+	builder := sdk.TransactionBuilderInit(sender, client)
+
+	package_id, _ := sdk.AddressFromHex("0x1")
+	module_name, _ := sdk.NewIdentifier("u64")
+	function_name, _ := sdk.NewIdentifier("sqrt")
+
+	builder.MoveCall(
+		package_id,
+		module_name,
+		function_name,
+		[]*sdk.PtbArgument{sdk.PtbArgumentU64(64)},
+		nil,
+		nil,
+	)
+
+	token := "test"
```

Comment:
Please move that next to the gasStationUrl and maybe call it `gasStationAuthToken` so it's clearer